scripts/ecs/appsignals/sample-app/task-definitions/spring-petclinic-billing-service.json

```diff
@@ -0,0 +1,182 @@
+{
+  "family": "billing-service",
+  "networkMode": "awsvpc",
+  "containerDefinitions": [
+    {
+      "name": "billing-service",
+      "image": "billing-service-image",
+      "essential": true,
+      "environment": [
+        {
+          "name": "DJANGO_SETTINGS_MODULE",
+          "value": "pet_clinic_billing_service.settings"
+        },
+        {
+          "name": "PYTHONPATH",
+          "value": "/otel-auto-instrumentation-python/opentelemetry/instrumentation/auto_instrumentation:/app:/otel-auto-instrumentation-python"
+        },
+        {
+          "name": "OTEL_EXPORTER_OTLP_PROTOCOL",
+          "value": "http/protobuf"
+        },
+        {
+          "name": "OTEL_TRACES_SAMPLER_ARG",
+          "value": "endpoint=http://localhost:2000"
+        },
+        {
+          "name": "OTEL_LOGS_EXPORTER",
+          "value": "none"
+        },
+        {
+          "name": "OTEL_PYTHON_CONFIGURATOR",
+          "value": "aws_configurator"
+        },
+        {
+          "name": "OTEL_TRACES_SAMPLER",
+          "value": "xray"
+        },
+        {
+          "name": "OTEL_EXPORTER_OTLP_TRACES_ENDPOINT",
+          "value": "http://localhost:4316/v1/traces"
+        },
+        {
+          "name": "OTEL_AWS_APPLICATION_SIGNALS_EXPORTER_ENDPOINT",
+          "value": "http://localhost:4316/v1/metrics"
+        },
+        {
+          "name": "OTEL_AWS_APPLICATION_SIGNALS_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "OTEL_RESOURCE_ATTRIBUTES",
+          "value": "service.name=spring-petclinic-demo-billing-service"
+        },
+        {
+          "name": "OTEL_METRICS_EXPORTER",
+          "value": "none"
+        },
+        {
+          "name": "OTEL_PYTHON_DISTRO",
+          "value": "aws_distro"
+        },
+        {
+          "name": "EUREKA_SERVER_URL",
+          "value": "discovery-server-cluster-name.ecs-petclinic"
+        },
+        {
+          "name": "BILLING_SERVICE_IP",
+          "value": "billing-service-cluster-name.ecs-petclinic"
+        },
+        {
+          "name": "DB_NAME",
+          "value": "postgres"
+        },
+        {
+          "name": "DB_USER",
+          "value": "djangouser"
+        },
+        {
+          "name": "DB_USER_PASSWORD",
+          "value": "asdfqwer"
```

Comment:
Is the password used? If yes, please hide it or let customers to decide it.

Reply:
Updated password to use randomly generated string.